app/src/main/java/org/openobservatory/ooniprobe/fragment/ResultListFragment.java

```diff
@@ -154,7 +154,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
                         .withMessage(getString(R.string.Modal_DoYouWantToDeleteAllTests))
                         .withPositiveButton(getString(R.string.Modal_Delete))
                         .build().show(getChildFragmentManager(), null);
-                CountlyManager.recordEvent("DeleteAllTests");
```

Comment:
Why was this moved?

Reply:
Because we want to record how many people actually delete the tests, not the ones who just press on the button (before modal)